docs/sources/send-data/otel/_index.md

```diff
@@ -124,8 +124,9 @@ otlp_config:
   # Configuration for Resource Attributes to store them as index labels or
   # Structured Metadata or drop them altogether
   resource_attributes:
-    # Configure whether to ignore the default list of Resource Attributes to be
-    # stored as Index Labels and only use the given Resource Attributes config
+    # Configure whether to ignore the default list of resource attributes set in
```

Comment:
To me this reads as: _"if this is set to true, we ignore `distributor.otlp.default_resource_attributes_as_index_labels`" and use whatever is configured in `attributes_config`_. Problem is, the `action` in `attributes_config` only support `structured_metadata` or `drop` so how can you define which attributes to store as index labels?

Reply:
Resolving. I was wrong.